File: wms/src/main/webapp/system/app/model/idata/dock_SheetModel.js
Ext.define('cms.model.idata.dock_SheetModel', {
    extend: 'Ext.data.Model',
    fields: [			
	       	{name:'rgstName'},
			{name:'startTime'},
			{name:'endTime'},
			{name:'TDock'},
			{name:'UDock'},
			{name:'FDock'},
			{name:'carsCount'}
    ],
    idProperty:'rgstName,startTime,endTime'
});

File: model/src/main/java/com/sealinkin/idata/model/DockSheetModel.java
package com.sealinkin.idata.model;

import com.sealinkin.idata.po.DockSheet;

public class DockSheetModel extends DockSheet{
	
	private static final long serialVersionUID = 1L;
	
	private String rgstName;
	private String startTime;
	private String endTime;
	public String getRgstName() {
		return rgstName;
	}
	public void setRgstName(String rgstName) {
		this.rgstName = rgstName;
	}
	public String getStartTime() {
		return startTime;
	}
	public void setStartTime(String startTime) {
		this.startTime = startTime;
	}
	public String getEndTime() {
		return endTime;
	}
	public void setEndTime(String endTime) {
		this.endTime = endTime;
	}
	
	

}
